pallets/proposals/src/tests.rs

```diff
@@ -1974,26 +1977,28 @@ fn withdraw_percentage_milestone_completed_refund_locked_milestone() {
 
         Proposals::refund(Origin::root(), project_key).unwrap();
 
-        //ensuring the refunded amount was transferred back successfully
-        let contributor_balance_pre_refund: u64 = 9_900_000;
-        assert_eq!(contributor_balance_pre_refund, Balances::free_balance(&bob));
-        assert_eq!(
-            contributor_balance_pre_refund,
-            Balances::free_balance(&charlie)
-        );
-
-        //contribute success event
         let exp_projectfundsrefunded_event = <frame_system::Pallet<Test>>::events()
             .pop()
             .expect("Expected at least one EventRecord to be found")
             .event;
         assert_eq!(
             exp_projectfundsrefunded_event,
-            mock::Event::from(proposals::Event::ProjectLockedFundsRefunded(
+            mock::Event::from(proposals::Event::ProjectFundsAddedToRefundQueue(
                 project_key,
                 800000u64
             ))
         );
+
```

Comment:
Can we add a test that covers multiple passes of the queue

say 10 contributors with `RefundsPerBlock` set to 2. Just to ensure we are keeping track of who has been refunded and don't accidentally double refund sometime in the future :)

Reply:
I have added a test that ensures only `T::RefundsPerBlock` number of contributers are refunded (given there is no space left in on_idle) and that the length of `RefundsQueue` decreases by `T::RefundsPerBlock` per block, this ensures there cannot be double refunds.